vyxal/elements.py

```diff
@@ -5858,6 +5859,14 @@ def separate_runl_encode(lhs, ctx: Context):
     return lengths
 
 
+@element("ø%", 1)
+def sha256_hash(lhs, ctx):
+    """Element ø%
+    ...
+    """
+    return sha256(str(lhs).encode()).hexdigest()
```

Comment:
A check for whether `lhs` is a string or not would be helpful, so users don't get weird errors saying `encode` can't be called on a `list` or whatever

Reply:
It converts to `str` anyway:

> `str(lhs).encode()`

Reply:
Ah, my bad, didn't notice you did `str(lhs)`